subt/docker/robotika/ros/get_robot_name.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
We don't need this file since we can get the robot name using `rosparam get /robot_names`.

Reply:
Ok. I will remove it.